client/src/App.tsx

```diff
@@ -41,7 +41,7 @@ function App(): JSX.Element {
                   <Route exact path="/forgot-password" component={ForgotPassword} />
                   <Route path="/reset-password/:token/:id" component={ResetPassword} />
                   <Route exact path="/artist" component={AboutArtist} />
-                  <Route exact path="/dashboard" component={Discovery} />
+                  <Route exact path="/dashboard" component={Dashboard} />
```

Comment:
I am pretty sure right now Discovery is what we want to render at the Dashboard route.

Reply:
:)., i just figured that when i saw minh making those changes., i will change it to discovery., thank you for pointing out